addons/website/__init__.py

```diff
@@ -37,8 +37,3 @@ def rem_website_id_null(dbname):
             ]).unlink()
 
```

Comment:
Well, IMO it needs to be kept - because it is an actual fix of another problem - which made the page rendering fail in that situation, because the view's XML refers to a field that was not loaded prior to the fix.